use-timescale/popsql.md

```diff
@@ -0,0 +1,154 @@
+  parameterize your queries or use `if` statements.
+
+<Procedure>
+
+## Get started
+
+To use PopSQL with Timescale:
+
+1.  **Open PopSQL from Timescale Console**
+
+    Log in to [Timescale Console](https://console.cloud.timescale.com/), then click `PopSQL`. 
+  
+    ![PopSQL button within Timescale
+    Console](https://assets.timescale.com/docs/images/popsql_button_in_console.png)
+
+    You may be prompted to sign into Timescale again.
+
+2. **Link your Timescale and PopSQL accounts** 
+
+   If you use the same email for existing Timescale and PopSQL accounts, these accounts are linked automatically. If you are using PopSQL for the first time, confirm that you want to create a PopSQL account. 
+
+    ![Agree to PopSQL terms](https://assets.timescale.com/docs/images/popsql_terms.png)
+
+   This creates a new PopSQL organization that is associated with your Timescale project.
+
+3. **Connect to a Timescale Service**
+
+    Select a service and enter your credentials, then click `Connect`.
+
+   ![Enter service credentials](https://assets.timescale.com/docs/images/popsql_enter_service_credentials.png)
+
+    If your Timescale service runs in a VPC, follow the instructions in [What if my service
+    is inside a VPC?](#what-if-my-service-is-within-a-vpc).
+
+4. **Run a test query**
+
+      You've successfully connected PopSQL to your Timescale service!
+      Try running a test query like `select current_date`.
+      
+      ![Running a test query in PopSQL](https://assets.timescale.com/docs/images/popsql_test_query.png)
+
+</Procedure>
+
+Now you have set up PopSQL, see how to easily do the following:
+
+- [Write a query](https://docs.popsql.com/docs/writing-a-query)
+- [Share a query with your
+  teammates](https://docs.popsql.com/docs/sharing-a-link-to-your-query-and-results)
+- [Create a chart from your
+  data](https://docs.popsql.com/docs/creating-charts)
+- [Create a dashboard of multiple query
+  results](https://docs.popsql.com/docs/creating-dashboards)
+- [Create schedules for your
+  queries](https://docs.popsql.com/docs/scheduled-queries)
+
+
+
+## FAQ
+
+### How do I access PopSQL?
+
+You connect from either:
+
+-  **Timescale Console**
+
+    Either:
+    - When you open Timescale Console, click `PopSQL`:
+    
+        ![PopSQL button within Timescale
+        Console](https://assets.timescale.com/docs/images/popsql_button_in_console.png)
+
+    - In Timescale Console, click  `Project settings`, then click `View Project in
+      PopSQL`. 
+    
+      ![Timescale project settings](https://assets.timescale.com/docs/images/popsql_project_settings.png)
+
+      This resyncs your Timescale project with your PopSQL organization. Any new services are added automatically.
+
+- **app.popsql.com**
+
+    Once you've connected your Timescale account to PopSQL, you can access PopSQL
+    directly at [app.popsql.com](https://app.popsql.com) or use the [desktop
```

Comment:
How does this work if you signed up via Timescale (i.e. you didn't have a prior PopSQL acocunt)? Since authentication happens with your Timescale account, what's authentication mechanism or password I should use?